\chapter[\textit{In situ} prokaryotic survey, quantification and bacterial bioluminescence\string: estimation at the ANTARES station]{\textit{In situ} prokaryotic survey, quantification and bacterial bioluminescence\string: estimation at the ANTARES station
\chaptermark{\textit{In situ} prokaryotic survey}}
\chaptermark{\textit{In situ} prokaryotic survey}
\minitoc
%\label{chap4}
\newpage

\section[Introduction\string: from laboratory to \textit{in situ} measurements]{Introduction\string:  from laboratory to \textit{in situ} measurements
\sectionmark{Introduction\string: from laboratory to \textit{in situ} measurements}}
\sectionmark{Introduction\string: from laboratory to \textit{in situ} measurements}

In Chapter 4, experimentations have been performed, in the laboratory, under controlled growth conditions and with a pure culture, using the bioluminescent bacterial strain \textit{Photobacterium phosphoreum} ANT-2200 as a model. This necessary first step gave us some clues, using experimental results, to validate or not the relevance of considering bioluminescent bacteria as inducers of the intense bioluminescence events detected at the ANTARES station, as we proposed in Chapters 2 and 3. Laboratory experiments were efficient to define independent effects of hydrostatic pressure, temperature and growth medium on oxygen consumption, growth and bioluminescence activity. We demonstrate that high hydrostatic pressure (22 MPa) compared to atmospheric pressure (0.1 MPa), increases bioluminescence activity for this bacterial strain, defined as moderately piezophile. Moreover, several physiological adaptations, better yield for both oxygen consumption and photon emission, have been highlighted under \textit{in situ} conditions compared to atmospheric ones. Furthermore, physiological aspects still need to be defined. \\

%\textit{In situ}, the bioluminescence peaks described using times series, in Chapter 1 and 2, at the ANTARES station in March 2009 and March 2010, reach several thousand of photons per second (maximal median rate about $8 \times 10^6$ Hz). At laboratory, in \cite{martini2013}, we described about $1.9 \times 10^{-2}$ photons s$^{-1}$ per cell at 13\degres C and 22 MPa (\textit{in situ} conditions) and in SWC medium. These values give an extrapolated number of bioluminescent cells about 4.2 $\times$ 10$^9$ to reach such \textit{in situ} light emission. This bacterial count can be related to volume unit as photomultipliers have been estimated to detect photon within a volume of about XXX m3. These estimations refer to a concentration about XXX cell mL$^{-1}$.\\
 
As far as we know, until the present study, milky sea were the only large phenomena of high bioluminescence activity detected \textit{in situ}. Such phenomenon has been observed by naked eye and by satellites. \cite{miller2005}, \cite{nealson2006} and \cite{lapota1988} attribute these phenomena to bioluminescent Bacteria and give an estimation for their concentration of about 2.8 $\times $ 10$^8$ cells cm$^{-2}$. However, these phenomena were described from satellite images or seamen opportunist observations and recorded at the surface of the oceans. Using neutrino telescope and, more generally, PMTs as bioluminescence detectors, such surprising events will be more easily observed, quantified, and explained even in extreme environments such as the dark ocean. To validate such an aim, the part of bacterial bioluminescence has to be quantified and differentiated from the total bioluminescence signal.\\

\begin{figure}[!h]
\linespread{1} 
\centering
\includegraphics[width=10cm]{milkysea.jpg}
\caption[Milky sea detection from satellite observation in the Indian Ocean.]{Milky sea detection from satellite observation in the Indian Ocean. The bioluminescent feature (lower right) is to scale, but has been colorized and enhanced, so it appears much brighter in relation to the scene than it would naturally. The glowing area has been estimated about 15,400 km$^2$ and attributed to bioluminescent Bacteria (\textit{Vibrio harveyi}) associated to \textit{Phaeocystis} (Haptophyte, Eukaryotes). From \underline{http://biolum.eemb.ucsb.edu/organism/milkysea.html}.}
\label{esp}
\end{figure}

%\newpage
%\section{Foreword}

In the next Chapter, we aim at answering the question of bioluminescent bacteria abundance, variability and their possible implication in the signal recorded from PMTs. As first objective, we describe the prokaryotic community over one-year survey at the ANTARES station. At the same time, discrete sampling for environmental variables (temperature, salinity, DOC, NO$_3^-$,PO$_4^{3-}$,Si(OH)$_4$, and dissolved oxygen) are performed during MOOSE and CASCADE cruises. Unfortunately, the stop of the IL07 instrumented line, at the end of 2010, does not permit to access to high frequency dataset for temperature and salinity. From these data, the bioluminescent bacterial contribution, its activity and variability over the year are quantified thanks to molecular biology methods and analyzed conjointly with environmental data obtained monthly by discrete profiles and water sampling. This sampling strategy involving instrumentation at sea, meteorology hazards have been one of the major constraint. However, from this preliminary work, presented as an "article in preparation", first results and perspectives will be proposed.\\

Conjointly to this survey, the CASCADE cruise was conducted in 2011, from March 1$^{st}$ to 30$^{th}$, in the Gulf of Lion and at the ANTARES site aiming at the description of deep convection phenomena. During this cruise, hydrological data show a winter convection in March 2011, reaching 1,600 m depth (X. Durieu de Madron pers. com., M. Boutrif PhD thesis 2012). Hence, this phenomenon observed in 2011 is much less intense and less deep than the previous ones described in Chapter 1, \citep{tamburini2013}, \ref{manuscrit1} reaching 2,300 m depth in 2010 (see Figure \ref{F4}) and consequently not directly impacting the ANTARES station.\\ 

%Ce Chapitre 4 a pour objectif de définir l'importance de l'émission de bioluminescence bactérienne dans le signal enregistré par les photomultiplicateurs, sur le site ANTARES.\\

%Un premier objectif de ce Chapitre est le suivi, au cours de l'année 2011, de la communauté procaryotique totale sur le site ANTARES. A partir de ce suivi, la part de Bactéries bioluminescentes, son activité ainsi que sa variabilité au cours du temps sont quantifiés. L'arrêt de la ligne océanographique câblée sur le site ANTARES, en fin d'année 2010, n'a pas permis l'utilisation des données environnementales échantillonnées à haute fréquence et en temps réel. Un échantillonnage discret a donc été réalisé au cours des campagnes MOOSE et CASCADE. Ces prélèvements, nécessitant des moyens à la mer, ont été contraints par les aléas météorologiques. Cependant, à partir de ce travail préliminaire, présenté sous la forme d'un manuscrit en préparation, de premiers résultats, une amélioration des protocoles ainsi que des perspectives seront proposées.\\

%Parallèlement à ce suivi, la campagne CASCADE s'est déroulée du 1$^{er}$ au 11 Mars 2011 dans les eaux du Golfe du Lion ainsi que sur le site ANTARES avec pour objectif l'étude des phénomènes de convections profondes. Au cours de cette campagne, les données hydrologiques collectées indiquent la formation d'une convection hivernale peu intense en Mars 2011 ne dépassant pas 1,600 m de profondeur (thèse M. Boutrif 2012, comm. pers. X. Durieu de Madron) et par conséquent, sans impact sur le site ANTARES. Ce phénomène observé en 2011 est sans commune mesure avec les données présentées au Chapitre 1, \citep{tamburini2013}, \ref{manuscrit1} décrivant une convection jusqu'à 2,300 m de profondeur au cours de l'année 2010 (voir Figure \ref{F4}).\\

\textit{This work has been granted by EC2CO-BIOLUX. C. Tamburini is the PI for the project. D. Lefèvre, M. Garel, C. Tamburini and me performed the sampling at sea at the ANTARES station during both MOOSE and CASCADE campains. L. Casalot and et S. Isart elaborated the beginning of the study with \textit{lux} gene sequence determination and preliminary work. V. Michotey and S. Guasco performed the experimentations and the interpretation of the results for molecular biology. I summarized, interpreted and wrote the results.}\\

%\textit{Ce travail a été financé par le projet EC2CO-BIOLUX, résultat d'une collaboration au sein du laboratoire MIO. Christian Tamburini est porteur du projet. Dominique Lefèvre, Marc Garel, Christian Tamburini et moi-même avons effectué les prélèvements en mer sur le site ANTARES au cours des campagnes MOOSE et CASCADE. Laurie Casalot et Stéphie Isart (stagiaire école d'ingénieur) ont élaboré les prémices de l'étude concernant la séquence du gène \textit{lux} utilisé. Valérie Michotey et Sophie Guasco ont effectué les manipulations ainsi que l'interprétation des résultats de biologie moléculaire. J'ai regroupé, écrit et traité les résultats présentés ci-après.}\\

\newpage
\section{Article 4}
\subsection[Temporal survey of prokaryotic communities, presence and activity of bioluminescent bacteria at the deep ANTARES station (North-western Mediterranean Sea)]{Temporal survey of prokaryotic communities, presence and activity of bioluminescent bacteria at the deep ANTARES station (North-western Mediterranean Sea).
\sectionmark{ARTICLE 4}}
\sectionmark{ARTICLE 4}

\begin{center}
%\Large{Temporal survey, presence and activity of bioluminescent bacteria at the deep ANTARES station\\ (North-Western Mediterranean Sea).}\\
Martini S.$^{1,2\ast}$, Michotey V.$^{1,2}$, Guasco S.$^{1,2}$, Casalot L.$^{1,2}$,\\ Garel M.$^{1,2}$, Tamburini C.$^{1,2}$
\end{center}
\vspace{20mm}

\noindent $^1$ Aix-Marseille Université,  Mediterranean Institute of Oceanography (MIO), 13288, Marseille, Cedex 09, France ; \\
$^2$ Université du sud Toulon-Var (MIO), 83957, La Garde cedex, France ; CNRS/INSU, MIO  UMR 7294; IRD, MIO UMR 235.\\
\null
\vspace{15mm}
\begin{center}
$\ast$ Corresponding author\string:  \underline{severine.martini@univ-amu.fr}\\
\end{center}
\vspace{32mm}
\textbf{Article in preparation} 

\newpage
\textbf{Abstract}\\

The ANTARES station is indirectly used as an observatory for the bioluminescence record in the deep (2,000 m depth) North-western Mediterranean Sea. In this study we describe the prokaryotic community, at the deep ANTARES site, over one-year survey in 2011. Relatively stable prokaryotic abundance and diversity have been recorded during the year with the detection of \textit{Chloroflexi} and \textit{Proteobacteria} as the most active members. Conjointly to these data, environmental conditions are characterized using water samplings (temperature, salinity, nutrients, dissolved organic carbon and oxygen). During this survey, no hydrological particularities affecting the ANTARES station have been observed with minor variability in temperature, salinity, nutrients, dissolved organic carbon and oxygen. Then, the final objective was to focus on bioluminescent-bacterial variability over the year and during a low bioluminescence event, recorded in the deep sea. Using the \textit{lux}F gene, about 0.1 to 1\% of bioluminescent bacteria have been detected in the sampling period. These bacteria were mainly belonging to \textit{Photobacterium} genus. Moreover, bioluminescent bacteria have been found to be mainly active cells even during a low bioluminescence event.

\newpage
\subsection*{Introduction}

Bioluminescence is the production of light by organisms as a result of a chemical reaction. In the ocean, this process is the primary source of light \citep{Haddock2010}. Bioluminescent organisms are spread within 14 phyla \citep{widder2002} covering eukaryotic and prokaryotic domains. Within evolution,  the distribution of bioluminescence ability does not appear to follow any obvious phylogenetic distribution. In the mesopelagic zone, from 500 to 1,000 m, \cite{herring2002} estimate that 90\% of deep-sea fish, 65\% of decapods, 20-30\% of copepods as well as ostracods are bioluminescent. This wide variety and abundance of species using bioluminescence as well as the pattern of light emission, decreasing with depth, have lead to the assumption that bioluminescence-activity records could be used as a relative proxy of the marine-organisms abundance \citep{moline2009,tokarev1999}.\\

In the literature, the \textit{in-situ} bioluminescence emission is mainly attributed to eukaryotic organisms such as dinoflagellates \citep{cussatlegras2005,widder2010}, or zooplankton \citep{craig2010}. Correlations have been shown, above 1,000 m depth, between the concentration of  chlorophyll-\textit{a} \citep{craig2010,lapota1989} or dinoflagellates \citep{piontkovski2003} and the bioluminescence emission \citep{Haddock2010}. Some studies assert that bioluminescence intensity reflects zooplankton abundance. In \cite{craig2010}, this relation was not supported below 1,000 m depth suggesting the possible implication of other organisms. Bioluminescent bacteria could explain part of this signal during special events of deep convection as suggested by \cite{tamburini2013}. However, bioluminescence from bacteria does not respond to mechanical stimulation. In most studies, bioluminescence is estimated by bathyphotometers using mechanical stimulation of bioluminescent eukaryotic organisms \citep{cussatlegras2007}. Consequently, the use of only bathyphotometers does not permit to detect bacterial bioluminescence. This technical approach probably leads to an underestimation of bacterial contribution to the bioluminescence activity.\\ 

Dark ocean represents about 95\% of the oceans and accounts for 55\% of all prokaryotes (Archaea and Bacteria) found in aquatic habitats \citep{whitman1998,aristegui2009,carlson2010}. Within the biological pump process, these organisms contributes to the carbon cycle by being active in the remineralization of organic matter in the deep sea \citep{aristegui2009,nagata2010,cho1988} and by feeding back nutrients and dissolved inorganic carbon by respiration. Amongst these deep-sea prokaryotes, bioluminescent bacteria attached to particles are known to play a role in the degradation of particulate carbon. However their \textit{in situ} importance is still unclear \citep{zarubin2012}. Described bioluminescent bacteria belong, so far, to the Gammaproteobacteria subclass and are affiliated to \textit{Vibrio}, \textit{Photobacterium} and \textit{Shewanella} genera \citep{kita2006}. Bioluminescence activity is driven by the product of the \textit{lux}ICDABFEG genes belonging to one operon.  For a cellular concentration, estimated at about 10$^7$ to 10$^8$ cells in liquid culture, bioluminescent genes are activated by quorum-sensing phenomenon \citep{nealson1970,eberhard1972}. Such high cellular concentration can be reached in symbiotic association, and most probably for bacteria attached to sinking particles or "marine snow" \citep{azam1998,alldredge1987}. At the surface, bioluminescent bacteria  have frequently been recorded. They can be responsible for large and intense emission of bioluminescence referred as "milky sea" \citep{miller2005,miller2006}. Bacterial bioluminescence has been intensively studied from punctual \textit{in-situ} samplings \citep{malave2010} or from experiments at laboratory \citep{hastings1983}. However, only few studies have dealt with \textit{in-situ} temporal dynamic of such bacteria, due to time-consuming and expensive reasons \citep{ruby1978,yetinson1979,orndorff1980}. Within these few studies, seasonal changes in bioluminescent bacteria have been recorded in surface layers and a correlation between \textit{Vibrio} \textit{harveyi} and the surface temperature has been demonstrated \citep{ruby1978}. \cite{yetinson1979} and \cite{orndorff1980} also investigate, over time, links between bioluminescent bacterial strains and environmental variables at the surface, such as salinity or temperature. In the deep layer of the ocean (below 1,000 m depth), less information on the \textit{in-situ} bioluminescence and presence of organisms is available due to technical constraints. \cite{gentile2009} recorded bioluminescent bacteria in  the deep Mediterranean Sea (Thyrrenian Sea), based on the \textit{lux}A-gene detection from environmental DNA. These authors found unexpected high number of \textit{lux}A lineages mainly belonging to the \textit{Photobacterium} cluster. Furthermore, at the deep Mediterranean ANTARES site (2,000 m depth), \cite{tamburini2013} proposed bioluminescent bacteria to be the main contributors to the intense bioluminescence activity recorded during periods of deep convection occurring in the Gulf of Lion. The presence, at this station, of bioluminescent bacterial strain (\textit{Photobacterium phosphoreum} ANT-2200), isolated during a previous event of deep convection and high bioluminescence activity, also supports this hypothesis \citep{alali2010,martini2013}.\\

The recent development of neutrino underwater telescope \citep{amram2000} gives access to long time series luminescent records in real time. Such structures use photomultiplier tubes (PMTs) as photon detector for high-energy particles. This instrumentation is used by oceanographers as a bioluminescence sensor \citep{tamburini2013,craig2010} with no direct mechanical stimulation (excepted from current speed), compared to bathyphotometers. In this study, we use the ANTARES neutrino telescope, in the North-western Mediterranean Sea, as one of those observatory possibly able to detect no only bioluminescence emissions from eukaryotes but also from bacteria. A one-year water sampling conjointly to that continuous luminescence record, at 2,000 m depth, were used (1) to analyze the composition of the global prokaryotic community at this site on qualitative, quantitative, activity aspects based on ribosomal information, (2) to investigate the bioluminescent bacterial counterpart using \textit{lux} genes, (3) and to estimate possible links between \textit{in-situ} bioluminescence activity and bioluminescent bacteria.\\

\subsection*{Methods}

\subsubsection*{Site location and continuous record of deep bioluminescence}
The ANTARES project (Astronomy with a Neutrino Telescope and Abyss environmental RESearch) developed, since the end of 2007, a deep-sea cabled observatory, in the North western Mediterranean Sea. The ANTARES site is located 40 km off the French Mediterranean coast (42\degres 48'N, 6\degres 10'E) at 2,475 m depth. At first, this site was dedicated to the search of high energy particles such as neutrino \citep{amram2000}, \citep{aguilar2007}. About 885 PMTs are located between 2,000 and 2,400 m depth for the purpose of particle Physics. All the 12 ANTARES mooring lines are connected, via an electro-optical cable to a shore station providing real-time acquisition. These PMTs sample photons emission at high frequency (0.013 s), and are connected to the ANTARES-transmission cable, delivering real-time data. The bioluminescence activity analyzed in this study is the mean rate of PMTs recorded at the ANTARES telescope, at 2,000 m depth, and sampled on a 7-days period around the sampling day (period from $n-3$ to $n+3$ with $n$ the water-sampling day). Current speed has been sampled using an Accoustic Doppler Current Profile 300 kHz RDInstruments.\\

\subsubsection*{Sample collection and on-field processing}
The monitoring strategy for the biogeochemical sampling has been established in the framework of the MOOSE and CASCADE programs. The ANTARES site has been visited in January, March, May, June, August and October 2011. A 12-Niskin-bottle rosette of 8 L was used to sample water at 2,000 m depth. These water samplings give access to\string: potential temperature, salinity, Dissolved Organic Carbon (DOC), nitrates (NO$_3^-$), phosphates (PO$_4^{3-}$), silicates (Si(OH)$_4$), and dissolved oxygen (O$_2$).\\

Potential temperature and salinity have been sampled from a CTD probe SBE 9+. The dissolved oxygen has been recorded using an oxygen sensor SBE 43 cross-calibrated by oxygen Winkler measurements \citep{Gaarder1927, Bryan1976,Williams1981}.\\

DOC content was determined  using a Shimadzu model TOC-V Total Carbon Analyzer with a quartz combustion column filled with 1.2\% Pt on silica pillows. Standardization of the instrument was performed daily using Milli-Q water as blank and potassium hydrogen phthalate diluted in Milli-Q water (range 0-125 mMC) prepared just before sample analysis as a standard \citep{sohrin2005}. The running blank injected in triplicate after every 4 samples corresponding to the peak area of the Milli-Q water acidified with H$_3$PO$_4$. The DOC  concentration was determined from 3 to 4 independent analyzes after subtracting the running blank. Low-carbon water (LCW) and deep-seawater reference (DSR) were kindly provided by the Bermuda Biological Station and were measured daily to monitor the accuracy and the stability of the TOC analysis. The nominal analytic precision of the analysis procedure was within 2\%.\\

The water samples, for NO$_3^-$, PO$_4^{3-}$, Si(OH)$_4$ determinations, were collected with Niskin bottles and preserved at -20\degres C until analysis, according to \cite{aminot2004,aminot2007}. Nutrient concentrations were determined colorimetrically with an automated method, using a semi automatic Technicon Autonanalyser II (detection limit=0.05 $\mu$moles L$^{-1}$), according to \cite{treguer1975}.\\

%Values for chlorophylle-\textit{a} are satellite data recorded at the sea surface using weekly average. These data are from MODIS algorithm \citep{letelier1996} and have been analyzed in collaboration with F. d'Ortenzio.\\
%For bacterial count, samples have been performed by cytometry, using the PRECYM platform and a sorting-cytometer Influx\textsuperscript{\textregistered} (Becton Dickinson). Samples are taken into 5 cm$^3$ cryotubes where 0.2 cm$^3$ of PFA are added to 1.8 cm$^3$ of seawater. Homogenized samples are kept at room temperature for 20 min before to be stored into nitrogen. SYBR Green or DAPI stains are used to differentiate cells from particles. DAPI stain is a DNA-fluorochrom  ($\lambda$ excitation = 351 nm, $\lambda$ emission = 461 nm), and SYBR Green, is a more general nucleic acid stain ($\lambda$ excitation = 490 nm, $\lambda$ emission = 520 nm). High level of Nucleic Acids (HNA) and Low level of Nucleic Acids (LNA) are supposed to be linked to active and inactive cells, respectively \citep{Lebaron2002}, \citep{Lebaron2001}, \citep{Servais2003}. Within the HNA class, HNA-low and HNA-high can be distinguished referring to normal and dividing cells. \\
%\subsubsection*{Discrete water sampling for biology}
%The prokaryotic survey has been conducted close to the ANTARES station. The site has been visited in January, March, May, June, August and October 2011 and water has been sampled with a Niskin-bottles at 2,000 m depth. 
For DNA and RNA analyzes, between 6 and 17 L have been gently filtered, using a peristaltic pump, on autoclaved 0.22 $\mu$m GPWP 47 mm filters (Millipore$^\copyright$). For RNA analyzes, filtration time was limited to 15 min resulting in a filtered volume from 0.4 to 1.6 L. All filters were stored with 1 mL of RNAlater (Sigma-aldrich$^\copyright$) into liquid nitrogen directly after filtration.\\

\subsubsection*{DNA and RNA extraction} 
DNA was extracted from filters using ultra clean water DNA kit (MO BIO\textsuperscript{\textregistered}, CA) according to the manufacturer's instructions. The DNA was stored at -20\degres C. Total RNA was extracted from 0.4 to 1.6 L of water collected by gentle filtration on 0.22 $\mu$m filter using RNeasy minikit (Qiagen, Germany). Putative traces of DNA were removed by the Turbo DNase-free digestion (AMBION\textsuperscript{\textregistered}, Austin, Texas). The RNA was stored at -80\degres C. Synthesis of  cDNA was performed from about 50 ng of RNA using   GOscript reverse transcriptase (Promega) and reverse primer of PCR for bacterial (907R) or archaeal (915R or 1100R) SSU RNA and \textit{lux}F gene (\textit{luxF} 572R).\\
 
\subsubsection*{Screening and quantification of communities} 
The quantification of bacterial, archaeal 16S rRNA and 16S rDNA gene and \textit{lux}F gene and mRNA copy numbers were determined by qPCR with Sso Advanced\textsuperscript{\texttrademark} SyberGreen Supermix using a CFX96 Real Time System (C1000 Thermal Cycler, Bio-Rad Laboratories, CA, USA) with a calibration curve using a plasmid harboring corresponding gene fragment. For bacterial 16S rRNA, quantification was performed with primer sets (GML5F-Uni516R) \citep{takai2000,michotey2012}. Archaeal 16S rRNA gene and cDNA were quantified with two sets of primer\string: 1100R-931f \citep{Einen2008} and 300-516 \citep{michotey2012}.\\ 

For bioluminescent bacteria, different candidate genes were tested (\textit{lux}A, \textit{lux}G, \textit{lux}F). From the literature, \textit{lux}A codes for two sub-units of the bacterial luciferase. The \textit{lux}G codes for a protein close to the flavin reductase enzym. As far as we know, the \textit{lux}F function has not been precisely determined; However, it is detected in all mesopelagic and bathypelagic species, known so far, suggesting a major role for deep bioluminescent species \citep{meighen1993}.\\ 

Three sets of degenerated primers for amplifying \textit{lux}A, \textit{lux}F and \textit{lux}G genes were designed by aligning sequences available in the databases. Eight sequences from marine bacteria were used for \textit{lux}F (AY849520, AY849502, AB367391, AY849504, DQ988874, AY849485, AY341064 and DQ790856). Eight sequences were used for \textit{lux}G (AB261992, AY341064, AY849485, DQ988874, AY849502, AB367391, AY849504 and AY849486). Twelve sequences were used for \textit{lux}A (KC332289, X58791, AB058949, AY456753, AY341064, DQ988874, AY849504, AY345887, AY849502, AY849520). The primers were chosen according to various characteristics\string: optimum size for PCR product between 150 and 200 bp (range 75-250 bp), 25 bp primers (range 20-30 bp), optimal $T_m$ at 60\degres C (range 57-63\degres C), GC\% between 30 and 70\% and no more than 3 identical consecutive nucleotides.\\

Classical PCR gave better results on \textit{Photobacterium phosphoreum} DNA and on environmental DNA with primer sets amplifying \textit{lux}F (\textit{lux}F-385F ATGTTACATGTCAATGTYAATGAGG, \textit{lux}F-527R AATTACCAGCAAGATTCGCTACAT) or \textit{lux}G (\textit{lux}G-179F AGTTACATGTTGGGAGTTCGGTAAA, \textit{lux}G-349R ATAACCCTGTACCTCCAGCIATAAG) compared to \textit{lux}A (\textit{lux}A-598F AARAAAGCICARATGGAACTITATAATG, \textit{lux}A-784R TATTGGTNGCATTIACGTAIGAITC), in consequence these primers were kept for qPCR tests. As qPCR parameters were much better for \textit{lux}F than \textit{lux}G (threshold templates number corresponding to 40 versus 4750, PCR efficiency corresponding to 101\% versus 61\%), \textit{lux}F was chosen to detect bioluminescent bacteria. The program consists of a denaturing step of 10 s, an hybridization step of 10 s at 57.3\degres C, with an elongation step of 10 s. At the end of the PCR reaction, the specificity of the amplification was checked from the first derivative of their melting curves and by electrophoresis analysis.\\ 

\subsubsection*{Molecular fingerprinting analysis of microbial community} 
Denaturing Gradient Gel Electrophoresis (DGGE) analysis was used to screen samples prior to barcoding experiment. Ribosomal fragment were amplified using the GML5F 907MR primer set for Bacteria \citep{Bonin2002,Goregues2005} and 344f  915R primer-set for Archaea \citep{Casamayor2002}. To improve resolution of DGGE analyzes, primers GML5F-GC or 344f-GC contained a 40-nucleotide GC-rich sequence at the 5' end. PCR amplifications were carried out in 20 $\mu L$ reaction mixtures containing 1-2 ng of template DNA, 1.5 mM MgCl$_2$, 0.2 mM of dNTP, 1.25 or 0.31 $\mu M$ of each primer for Bacteria and Archaea respectively, and 1 U of Hot start polymerase plus 1 $\mu L$ of solution Q (Qiagen, Germany). The thermal cycling programs were similar to those previously described \citep{Casamayor2002,michotey2012}. DGGE was performed using a D-code Universal Mutation Detection System (Bio-Rad Laboratories Inc.). Equal amounts of PCR products were loaded in each lane ($\sim$300 ng SSU rRNA gene fragment). The 1 mm thick, 6\% (wt. /vol) polyacrylamide gels presented a denaturing gradient of 30\%-50\% for SSU rRNA gene fragment analysis.\\

Pyrosequencing of bacterial cDNA ribosomic gene of May and October samples were performed on fragment generated with the 16S universal Eubacterial primers (bact 343F, bact 806R) by MR DNA (Texas, USA). A single-step PCR using HotStarTaq Plus Master Mix Kit (Qiagen, Valencia, CA) were used under the following conditions\string: 94\degres C for 3 minutes, followed by 28 cycles of 94\degres C for 30 seconds; 53\degres C for 40 seconds and 72\degres C for 1 minute; after which a final elongation step at 72\degres C for 5 minutes was performed.  Following PCR, amplicon products were purified using Agencourt Ampure beads (Agencourt Bioscience Corporation, MA, USA). Samples were sequenced utilizing Roche 454 FLX titanium instruments and reagents according to manufacturer's instructions. Sequence data derived from the sequencing process were processed using a proprietary analysis pipeline (www.mrdnalab.com, MR DNA, Shallowater, TX). Sequences were depleted of barcodes and primers. Short sequences < 200 bp sequences with ambiguous base calls, and sequences with homopolymer runs exceeding 6 bp were removed. Subsequently a denoising step and chimeras check were performed. Operational Taxonomic Units (OTU) were defined after removal of singleton sequences, clustering at 3\% divergence (97\% similarity) \citep{dowdcallaway2008,dowdsun2008,edgar2010,capone2011,dowd2011,eren2011,swanson2010}. OTUs were then taxonomically classified using BLASTn against a curated GreenGenes database \citep{desantis2006} and ambiguous affiliations on species level were checked manually and compared with environmental clones.\\

\section*{Results and discussion}

\subsection*{Environmental conditions at the ANTARES station}

During the year 2011, potential temperature, salinity, Dissolved Organic Carbon (DOC), NO$_3^-$, Si(OH)$_4$, and oxygen concentrations have been sampled over vertical profiles, using discrete water sampling, at the ANTARES station (see Figure \ref{profiles}).  Continuous profiles have been represented, over the water depth, for each variable and smoothed, using a local polynomial regression (loess method \citealt{cleveland1991}) based on dataset (Figure \ref{profiles}). Then, the annual mean values at 2,000 m depth are summarizes (Table \ref{dataenvi}). Only low variations are detected over the water column, as well as at 2,000 m depth suggesting a stability of environmental conditions. In comparison with other bathypelagic regions already studied in the literature \citep{nagata2010}, the deep-sea waters at the ANTARES station (NW Mediterranean Sea) present higher temperature (12,896 / 0.4-5.4), lower nitrate (8.1 / 16-43), phosphate (0.35 / 1-3.2) and silicate concentrations (8.1 / 10-177). The recorded values at the ANTARES station are in the range of already observed ones in the deep Mediterranean Sea \citep{santinelli2010,Hansell2001,Hansell2009}.\\

\begin{figure}[!h]
\linespread{1} 
\centering
\includegraphics[width=14cm]{profils.pdf}
\caption{Vertical profiles for temperature, salinity, DOC, NO$_3^-$, PO$_4^{3-}$, and Si(OH)$_4$ using water sampling. Dots are samples values from the 6 sampling times, continuous lines are the profile using a local polynomial regression (loess method \citealt{cleveland1991}) and dotted lines are the standard error for each profile.}
\label{profiles}
\end{figure}

The Mediterranean Sea has long been described as a low nutrient concentration basin. Measurements of C\string:N\string:P ratios are useful to describe the functioning of the ecosystem and the 105\string:16\string:15\string:1 Redfield ratios proposed in the North-Atlantic for C\string:N\string:Si\string:P are still used as a reference \citep{Redfield1963,sterner2008,arrigo2004} to examine these changes in nutrient limitation. During this survey, at the ANTARES station, the N\string:P ratio varies between 21.6 and 26.6 and the Si\string:N ratio is stable between 0.9 and 1 (see Table \ref{dataenvi}). The deep Western Mediterranean basin (under 1,000 m depth) is characterized by nutrient ratios different from the Redfield ratio and are estimated at about N\string:P$\sim$20 and Si\string:N$\leq$1.0 \citep{santinelli2010,moutin2002}.\\

 \begin{table}
\center
\caption{Average and standard deviation for the main environmental variables over the year 2011, sampled at 2,000 m depth. T = potential temperature. S = salinity. DOC = Dissolved Organic Carbon. }
\begin{tabular}{lllllll}
\textbf{T} &\textbf{S}& \textbf{DOC}&\textbf{NO$_3^-$}&\textbf{PO$_4^{3-}$}& \textbf{Si(OH)$_4$}& \textbf{Oxygen}\\
 \textbf{(\degres C)} &\textbf{-}& \textbf{($\mu$mol L$^{-1}$)}&\textbf{($\mu$mol L$^{-1}$)}&\textbf{($\mu$mol L$^{-1}$)}& \textbf{($\mu$mol L$^{-1}$)}& \textbf{(mol kg$^{-1}$)}\\
 \hline
  \textbf{12.896} &\textbf{38.474}& \textbf{49.2}&\textbf{8.1}&\textbf{0.35}& \textbf{8.1}& \textbf{202.5}\\
  \textbf{(0.003)} &\textbf{(0.003)}& \textbf{(4.8)}&\textbf{(0.7)}&\textbf{(0.03)}& \textbf{(0.7)}& \textbf{(3.1)}\\
\end{tabular}
\label{dataenvi}
\end{table}
\vspace{5mm}

CTD profiles, for both temperature and salinity, give information on water-masses modifications. Using these set of data, a Theta-S diagram (Figure \ref{thetaS}) allows to identify hydrological changes for the different sampling times, below 1,200 m depth (and until 2,400 m depth). This representation highlights relatively stable water masses with however an intense hydrological event in March (pink dots) due to incursions of water masses from the surface and spreading to the ANTARES site (De Madron, pers. com.). However, these modifications of water characteristics returned to the previous stable state in May (orange dots). Water depth sampled for microbiological studies (2,000 m depth) were highlighted using bigger dots in Figure \ref{thetaS}. This shows that the water masses modifications clearly occur above 2,000 m, i.e above our sampling depth and above the ANTARES telescope, with no impact on the ANTARES telescope. \\

\begin{figure}[!h]
\linespread{1} 
\centering
\includegraphics[width=9cm]{theta-S.jpeg}
\caption{Potential temperature versus salinity diagram of CTD time series at the ANTARES site in January, March, May, June, September, and October 2011. Data shown are above 1,200 m. Colored dots represent data point sampled at 2,000 m depth. Changes in water masses occur at depth below 2,000 m with a come back to the stable state in May 2011.}
\label{thetaS}
\end{figure}
 
%During the sampled period, seasonal spring bloom can occur at the surface and involve carbon input into deep layers, to observe such phenomena, chlorophyll-\textit{a} records have been investigated. The Mediterranean Sea is described as an oligotrophic basin with an East-West gradient in chlorophyll-\textit{a} concentrations. In Figure \ref{chlorophyll}, chlorophylle-\textit{a} time series at the ANTARES station are recorded from satellite data and attributed to phytoplankton abundance. In March, a seasonal spring bloom occurred with a clear peak in chlorophyll-\textit{a}. These concentrations vary from 0.4 at the beginning of March to 1.2 $\mu$g dm$^3$ in the middle of March. However, the water sampling in this study has been performed few days before and 1.5 month after this spring phytoplanktonic bloom. Looking at the same time-scale as discrete water sampling for environmental variables, the 1$^{st}$ of March and the 22$^{nd}$ of May, a decrease is observed for chlorophyll-\textit{a} values in Figure \ref{chlorophyll}. After the occurrence of spring bloom at the surface, this organic mater is sinking with a high degradation rate by organisms over the water colum \citep{crispi2001}. This downward flux of suspended mater (MES) and Particular Organic Carbon (POC) is known to enhance higher bacterial production \citep{azam2001} into both the mesopelagic \citep{cho1988} and bathypelagic zones (for supporting data, see supplementary data, Figure \ref{resp_anne}).\\

%\begin{figure}[!h]
%\linespread{1} 
%\centering
%\includegraphics[width=14cm]{chlorophylle.pdf}
%\caption[Chlorophyll-\textit{a} time series recorded at the ANTARES station from satellite data, data acquisition.]{Chlorophyll-\textit{a} time series recorded at the ANTARES station from satellite data, data acquisition in collaboration with F. D'Ortenzio. Grey bars represent the discrete water sampling.}
%\label{chlorophyll}
%\end{figure}

%\begin{table}
%\center
%\caption{Variability of the major environmental variables over the year 2011, months corresponding to the minimal and maximal values are specified in brackets. DOC = Dissolved Organic Carbon.}
%\begin{tabular}{lllll}
%\textbf{Variable} &\textbf{Min value}& \textbf{Mean}&\textbf{Max}&\textbf{SD}\\
%\hline
%\textbf{Temperature (\degres C)}& 12.890 (June) & 12.897& 12.905 (Oct.) & 0.005\\
%\textbf{Salinity}& 38.470 (June) &38.475 & 38.480 (Oct.)& 0.004 \\
%\textbf{DOC ($\mu mol.dm^3$) }& 43.4 (May) & 48.3 & 53.2 (March) & 3.8\\
%\textbf{Nitrate ($\mu mol.dm^3$) }& 5.36 (Jan.)& 7.48 & 8.53 (May) & 1.21\\
%\textbf{Phosphate ($\mu mol.dm^3$)}& 0.300 (Jan.)& 0.325 & 0.355 (Oct.)& 0.02\\
%\textbf{Oxygen ($mol.kg^{-1}$)} & 199.0 (Aug.)& 202.6 & 206.1(Jan.) & 3.2\\
%\end{tabular}
%\label{}
%\end{table}

Consequently, during this survey, environmental conditions at the deep ANTARES station can be defined as relatively stable in nutrients, and without water mass input.\\

\subsection*{Annual dynamic of the prokaryotic community in the deep Mediterranean Sea}

The prokaryotic community was analyzed during this one-year-survey through ribosomal information (Figure \ref{qPCR1} A and B).\\

\begin{figure}[!h]
\linespread{1} 
\centering
\includegraphics[width=11cm]{qPCR1.pdf}
\caption[Quantitative estimation of A) 16S archaeal DNA (black bars) and RNA (grey bars) B) 16S bacterial DNA (black bars) and RNA (grey bars) among the year 2011, at the ANTARES station.]{Quantitative estimation of A) 16S archaeal DNA (black bars) and RNA (grey bars) B) 16S bacterial DNA (black bars) and RNA (grey bars) among the year 2011, at the ANTARES station (2,000 m depth). NA\string: no sampling for RNA. }
\label{qPCR1}
\end{figure} 

Archaeal 16S rDNA appeared not to be abundant ranging from 0.4$\times 10^2$ to 4.7$\times 10^2$ genes mL$^{-1}$ (Figure \ref{qPCR1} A). To investigate the possibility of underestimation due to primers selectivity, another primer set (primer 931F-1100R) was used for qPCR quantification. Results obtained with the second primer set were in good accordance with the previous one (see Table \ref{supqPCR}) and confirmed the low values obtained. These values are extremely low for deep-sea environment. Indeed, it has been previously observed that Archaea were nearly as abundant as Bacteria in meso- and bathypelagic systems \citep{church2003,herndl2005,karner2001,tamburini2009,teira2006} with about 10$^4$ cells mL$^{-1}$. At the ANTARES station, in April 2005, \cite{alali2010} refer to values of about 5 $\times$ 10$^4$ cells mL$^{-1}$ and about 40\% of DAPI counts belonging to Archaea  detected by CARD-FISH method. At this stage, an underestimation due to cell lysis problem could not be excluded. However, methodological differences between CARD-FISH method and qPCR are noticed and can possibly modify the archaeal detection between those methods. Then, the activity of this community was estimated by 16S rRNA abundance. A weak activity with very few archaeal 16S rRNA was detected and rRNA/rDNA ratios ranging from 0.03 to 0.35 corresponding to October and June respectively (Figure \ref{qPCR1} A).\\

\begin{sidewaystable}
\center
\caption{Values of molecular analysis survey in 2011 at the ANTARES station for both Archaea and Bacteria. ND\string:  Not Determined.}
\small
\begin{tabular}{lllllll}
\textbf{} &\textbf{January}& \textbf{March}&\textbf{May}&\textbf{June}&\textbf{August}&\textbf{October}\\
\hline
\textbf{Bacterial 16S rDNA}& 13.35 & 0.69 & 7.53 & 0.68 & 6.24 & 7.06\\
gene mL$^{-1}$ $\times 10^4$ & ($\pm$ 0.31) &($\pm$ 0.05)&($\pm$ 0.24)&($\pm$ 0.03)&($\pm$ 1.34)&($\pm$ 0.08)\\
&&&&&&\\
\textbf{Archaeal 16S rDNA} & 4.25 & 0.76 & 2.81 & 0.38 & 2.34 & 3.78 \\
gene mL$^{-1}$ $\times 10^2$ & ($\pm$ 0.06) &($\pm$ 0.48)&($\pm$ 0.20)&($\pm$ 0.02)&($\pm$ 0.20)&($\pm$ 1.23)\\
(Primer 300F-516R)&&&&&&\\
&&&&&&\\
\textbf{Archaeal 16S rDNA}& 4.72 & 0.77 & 2.42 & 0.28 & 2.90 & 2.73 \\
gene mL$^{-1}$ $\times 10^2$ & ($\pm$ 0.29) &($\pm$ 0.19)&($\pm$ 0.16)&($\pm$ 0.01)&($\pm$ 0.1)&($\pm$ 0.49)\\
(Primer 931F-1100R)&&&&&&\\
&&&&&&\\
\textbf{Bacterial 16S rRNA}& 31.95 & ND & 0.46 & 0.13 & ND & 4.00\\
gene mL$^{-1}$ $\times 10^4$ & ($\pm$ 5.40) &ND&($\pm$ 0.04)&($\pm$ 0.01)&ND&($\pm$ 0.07)\\
&&&&&&\\
\textbf{Archaeal 16S rRNA} & 0.258 &ND & 0.093 & 0.134 & ND & 0.104 \\
gene mL$^{-1}$ $\times 10^2$ & ($\pm$ 0.005) &ND&($\pm$ 0.002)&($\pm$ 0.018)&ND&($\pm$ 0.018)\\
&&&&&&\\
\textbf{\textit{Lux} DNA}& 1.4& 2.6& 1.8 & 5.0 & 7.2 & 72.0\\
gene mL$^{-1}$ $\times 10^2$ & ($\pm$ 0.2) &($\pm$ 2.9)&($\pm$ 1.2)&($\pm$ 2.2)&($\pm$ 7.7)&($\pm$ 5.3)\\
&&&&&&\\
\textbf{\textit{Lux} mRNA}& 5.5& ND& 7.7 & 7.0 & ND & 15.4\\
gene mL$^{-1}$ $\times 10^{-2}$ & ($\pm$ 0.1) &ND&($\pm$ 0.1)&($\pm$ 0.3)&ND&($\pm$ 7.0)\\
&&&&&&\\
%\hline
%\textbf{\textit{lux} mRNA /}&&&&&&\\
%\textbf{Bacterial rRNA }&0.2&ND&16.7&53.8&ND&3.8\\
%\textbf{ratio}&&&&&&\\
%\hline
\end{tabular}
\label{supqPCR}
\end{sidewaystable}


Abundance of bacterial 16S rDNA was higher than archaeal counterpart and varied from 0.7$\times 10^4$ to 13.3$\times 10^4$  genes mL$^{-1}$ (Figure \ref{qPCR1} B and Table \ref{supqPCR}). These results for bacterial 16S rDNA quantification are in good accordance with previous data obtained using FISH method at this station (\citealp{alali2010} and see references above). As for Archaea, the bacterial activity was estimated by 16S rRNA abundance. Bacteria appeared more active than Archaea although overall rRNA/rDNA ratios were not very high ranging from 0.06 to 2.39 in May and January respectively (Figure \ref{qPCR1}).\\ 

\begin{figure}[!h]
\linespread{1} 
\centering
\includegraphics[width=15cm]{phylo_archa.pdf}
\caption[A) Denaturing Gradient Gel Electrophoresis (DGGE) analysis estimation of 16S archaeal rDNA among the year 2011, at the ANTARES station. B) Phylogenetic tree of the archaeal 16S rDNA gene sequence from the clones DGGE bands.]{A) Denaturing Gradient Gel Electrophoresis (DGGE) analysis estimation of 16S archaeal rDNA among the year 2011, at the ANTARES station. Blue dot (\# 14) represents the sampled band in May and red dot (\# 17) the sampled band in October. \# 16, \# 19 and \# 15, \# 18 are similar sequences sampled into two different DGGE bands. B) Phylogenetic tree of the archaeal 16S rDNA gene sequence from the clones from DGGE bands. Values at nodes represent bootstrap value using neighbor joining. Scale bar denotes 0.02 substitution per nucleotide position.}
\label{arbre1}
\end{figure}

The diversity of total and active prokaryotes has been firstly analyzed by DGGE. Archaeal 16s rDNA DGGE profiles present similar patterns over the year 2011 (Figure \ref{arbre1}). Five clear bands are visible on the gel and sequences from the 3 most intense ones were obtained. These organisms are affiliated to the Euryarchaeaota  phylum in marine groups 2 and 3 cluster (Figure \ref{arbre1} B). The low quantity of archaeal 16S rRNA does not allow enough PCR material to analyze active archaeal community using DGGE.\\

 \begin{figure}[!h]
\linespread{1} 
\centering
\includegraphics[width=7cm]{DGGE_Bact.pdf}
\caption[Denaturing Gradient Gel Electrophoresis (DGGE) analysis of A) 16S bacterial rDNA and B) 16S bacterial rRNA during the year 2011, at the ANTARES station.]{Denaturing Gradient Gel Electrophoresis (DGGE) analysis estimation of A) 16S bacterial rDNA and B) 16S bacterial rRNA among the year 2011, at the ANTARES station. No sampling for 16S rRNA have been performed in March and June. }
\label{DGGE1}
\end{figure}

\begin{figure}[!h]
\linespread{1} 
\centering
\includegraphics[width=12cm]{arbre16S.pdf}
\caption[Phylogenetic tree of the bacterial 16S rDNA gene sequence from the clones DGGE bands.]{Phylogenetic tree of the bacterial 16S rDNA gene sequence from the clones DGGE bands. Values at nodes represent bootstrap values using neighbor joining. Scale bar denotes 0.02 substitution per nucleotide position.}
\label{arbre2}
\end{figure}

DGGE profiles of bacterial 16S rDNA show a major band (\# S1) at all sampling periods (Figure \ref{DGGE1} A), surrounded by numerous other fainter bands. The sequence of this intense band fell within \textit{Chloroflexi} phylum (Figure \ref{arbre2}), and is closely related to \textit{Chloroflexi bacterium} SCGC AAA240-O15, belonging to SAR 202 group, retrieved at 770 m depth at the HOT ALOHA station, subtropical ocean  gyre (99\% similarity, HQ975645, \citealp{swan2011}). \textit{Chloroflexi bacterium} was also observed over the water column, between 200 and 4,000 m at the ALOHA station \citep{delong2006} and in other mesopelagic and bathypelagic areas of various oceans \citep{nagata2010}. Moreover, concentration of this phylum has been observed to increase over the water column (mainly for SAR202, \citealp{varela2008}). Since qPCR results indicate that most of the cells are inactive, the diversity of the active members was investigated firstly by DGGE on RT-PCR fragment (see Figure \ref{DGGE1} B). About 8 intense bands are visible with similar patterns for January, May and June samples, and slightly different for October sample presenting less contrasted bands. The most intense band (\# S1) previously observed on DNA profiles also appear on all RT-RNA counterparts. This observation suggests a highly active strain in abundance all year round. Other active members visible although the corresponding bands on DNA DGGE gel show relatively lower intensity, such as \# S7 and \# S8 (identified as Gammaproteobacteria). In order to further investigate the diversity of active member, pyrosequencing was performed on cDNA of 16S rRNA of May and October samples, presenting different DGGE patterns.\\

\begin{figure}[!h]
\linespread{1} 
\centering
\includegraphics[width=14cm]{chap4_diag.pdf}
\caption[A) Venn diagram representation of total number of OTUs in October and May. B) Number of sequences per OTU for May and October samples showing a higher diversity for October than May.]{A) Venn diagram representation of total number of OTUs in October (110 OTUs) and in May (71 OTUs) and OTUs shared in between. About 50 OTUs (67\% of  sequences) are shared between those two samples. The Venn diagram depicts the percentage of present sequences. B) Number of sequences per OTU for May and October samples showing a higher diversity for October than May with 3 major OTU in May sample.}
\label{venn}
\end{figure}

For May and October samples 662 and 735 sequences were obtained respectively, spread in 133 OTUs with 3\% divergence cutoff. Their lengths vary between 200 and 480 nucleotides but are mainly about 400 nucleotides long. October sample appear more diverse with 110 OTUs detected in contrast to 71 for May (Figure \ref{venn} A). In accordance with DGGE analysis, May sample presents structured active community with 3 OTUs of major abundance whereas for October, the abundance within OTU is less contrasted (Figure \ref{venn} B). Amongst both samples, 50 OTUs are shared accounting for  67\% of all sequences (Figure \ref{venn} A). The affiliation of these OTUs showed that most of them belong to \textit{Chloroflexi} and \textit{Proteobacteria} with percentages reaching 58 and 33\% for May and 37 and 46\% for October, respectively (Figure \ref{esp}). The \textit{Chloroflexi} sequence identified as major member on DGGE analysis is also found by pyrosequencing as one of the abundant OTU (data not shown). Beside these phyla, the lower abundant OTUs fell in \textit{Bacteroides}, \textit{Cyanobacteria}, \textit{Synergistetes}, \textit{Nitrospirare}, \textit{Firmicutes}, \textit{Planctomycetes}, \textit{Actinobacteria} phyla, and candidate division ksb1 (Figure \ref{esp}).\\

\begin{figure}[!h]
\linespread{1} 
\centering
\includegraphics[width=9cm]{chap4_esp.pdf}
\caption[Diversity percentages representation among major phyla for October and May samples.]{Diversity percentages representation among major phyla for October and May samples. \textit{Chloroflexi} and \textit{Proteobacteria} are the main contributors to the diversity at the ANTARES station for those two samples.}
\label{esp}
\end{figure}

\subsection*{Quantification and dynamic of bioluminescent bacteria in a deep environment}

In parallel, current speed (Figure \ref{biolu} A) and bioluminescence activity (Figure \ref{biolu} B) were recorded continuously through the ANTARES telescope.\\

\begin{figure}[!h]
\linespread{1} 
\centering
\includegraphics[width=10cm]{TS_biolu3.pdf}
\caption[A) Current speed time series B) Bioluminescence time series recorded by photomultiplier tubes at the ANTARES station over the year 2011.]{A) Current speed time series B) Bioluminescence time series recorded by photomultiplier tubes at the ANTARES station over the year 2011.}
\label{biolu}
\end{figure}

Relatively low bioluminescence activity (Figure \ref{biolu} B) was observed in comparison to data previously monitored at that site \citep{tamburini2013}. It is worth noting that, at sampling times, the recorded current speeds were low with values of about 7 cm s$^{-1}$. In consequence, bioluminescent eukaryotes that emit light under mechanical stimulation are not stimulated and their contribution to the signal is probably not an important part of the signal. This observation might make easier the determination of bacterial bioluminescence within the signal. During these periods of low bioluminescence and without extreme events of water-mass input, the abundance of bioluminescent bacteria was investigated based on \textit{lux} genes detection, to monitor bioluminescent bacteria. Indeed, from ribosomal analysis, no genera are known to harbor bioluminescent bacteria and that it is known from the literature that this character does not fit with phylogenetic affiliation \citep{Haddock2010}.\\ 

%Between March and May 2011, a weak decrease in median bioluminescence activity (from 284 to 101 kHz) is observed in Figure \ref{biolu}. 

%Moreover, it has to be noticed that in the literature, deep-sea bioluminescent bacteria can be found, in the water column, as free living, attached to sinking particles or released by symbionts on fecal pellets. Due to quorum sensing \citep{hmelo2011,nunes2003}, a concentration of about 10$^8$ to 10$^9$ bioluminescent bacteria per mL$^{-1}$ has been estimated to involve the bioluminescence reaction. At this density, autoinductors concentration involves the chemical reaction for light emission. Such concentration can not be reached by free living bacteria, however, symbiotic or attached ones are possibly bioluminescence emitters. The \textit{lux} gene concentration has been detected at concentration up to 15.4 $\times$ 10$^4$ mL$^{-1}$. Consequently, bioluminescent bacteria are probably not directly linked to this low signal.\\

While most of the year, the abundance of \textit{lux} gene attained around 10$^2$ genes mL$^{-1}$ (Table \ref{supqPCR}) it corresponds to 0.1-1\% of the bacterial community (\textit{lux} gene DNA/bacterial 16S rDNA ratio) most of the year. In October sample, \textit{lux} genes concentration increased and reached 7.2$\times$ 10$^3$ genes mL$^{-1}$. The activity of \textit{lux} bacteria has been estimated by \textit{lux} RNA quantification. Their concentrations were in the same range than that of the DNA counterpart (about 10$^2$ \textit{lux} mRNA mL$^{-1}$) showing the bioluminescence activity for some of them. Bioluminescent bacteria seemed more active than the overall bacterial community since \textit{lux} RNA/DNA ratio (range 1.4 to 4.2) was higher than the 16S bacterial counterpart (0.06 to 2.39 range). \textit{Lux} population did not follow similar trend as the overall bacterial community. The highest specific activity estimated by RNA/DNA ratio (ratio of 4.2) was observed in May (Figure \ref{qPCR2}) whereas it corresponds to January for 16S bacterial ratio (ratio value of 2.39), in Figure \ref{qPCR1}.\\

\begin{figure}[!h]
\linespread{1} 
\centering
\includegraphics[width=10cm]{qPCR2.pdf}
\caption[Quantitative estimation of \textit{lux}F genes for bacterial DNA (black bars) and RNA (grey bars) among the year 2011, at the ANTARES station.]{Quantitative estimation of \textit{lux}F genes for bacterial DNA (black bars) and RNA (grey bars) among the year 2011, at the ANTARES station. NA\string: no sampling for RNA. }
\label{qPCR2}
\end{figure} 

Despite the small size of the \textit{lux} RT-qPCR fragment, sequences obtained in May and October, permit to affiliate the harboring organisms to \textit{Photobacterium} genera since all sequences fell in \textit{lux}F cluster including \textit{P.  phosphoreum}, \textit{kishitanii} and \textit{leiognathi} (Figure \ref{arbrelux}). Interestingly, this observation is similar to data reported in \cite{gentile2009} in Tyrrhenian Sea, at 2,750 m depth and for which numerous \textit{lux}A gene fell in \textit{Photobacterium} cluster. Furthermore, a \textit{Photobacterium} strain was also isolated at the ANTARES station \citep{alali2010}.\\

\begin{figure}[!h]
\linespread{1} 
\centering
\includegraphics[width=11cm]{arbreluxf.jpg}
\caption[Phylogenetic tree of the \textit{lux} genes sequence from the clones DGGE bands.]{Phylogenetic tree of the \textit{lux} genes sequence from the clones DGGE bands. Values at nodes represent bootstrap value using neighbor joining. Scale bar denotes 0.01 substitution per nucleotide position. Blue dots are selected band from May sample and blue ones are selected from October sample.}
\label{arbrelux}
\end{figure} 

Finally, we estimate, possible links between bioluminescent bacteria and bioluminescence total activity. In Figure \ref{boxplotbiolu}, bioluminescence activity has been integrated over 7 days and active bioluminescent bacteria are described using a \textit{lux}mRNA / bacterial rRNA ratio. Using data from PMTs, integrated during water sampling periods, bioluminescence activity is relatively low with only few variations over the year. Bioluminescence recorded in January and October are very close to background and higher values were observed for the other months (March, May, June and August). March and August samples correspond to highest values whereas no RNA data was available for these sample. Over the 4 available samples \textit{Lux} mRNA/bacterial 16S rRNA ratios ranged from 0.2 $\times$ 10$^{-2}$ to 53.8 $\times$ 10$^{-2}$. The two highest ratios (16.7 $\times$ 10$^{-2}$ and 53.8 $\times$ 10$^{-2}$) were obtained for May and June samples respectively. The lack of replicate due to technical reason did not allow to perform statistical tests. However, it should be noticed that similar trend is observed between bioluminescence activity and \textit{lux} mRNA/bacterial 16S rRNA ratios.\\ 

%For prokaryotic investigations, there are only low variations in DNA and RNA \textit{lux} genes number over the year and higher values in October whereas the bioluminescence signal recorded by PMTs at the ANTARES station show higher values in March and August (median values about 300 kHz) and extremely low in October (40 kHz). Therefore, it is difficult to relate these two dataset. For the complete bioluminescence time-series recorded in 2011, a high bioluminescence event is detected reaching 3,899 kHz (median 109 kHz for the whole time series) on February 2$^{nd}$ 2011. At the same time, current speed reach up to XX, such high current-speed intensity can mechanically stimulate eukaryotic bioluminescent organisms.\\

\begin{figure}[!h]
\linespread{1} 
\centering
\includegraphics[width=11cm]{bioluvsproka.pdf}
\caption[Box-and-whisker plot of bioluminescence 7 days around water sampling periods and \textit{lux} mRNA/bacterial rRNA ratio.]{Box-and-whisker plot of bioluminescence 7 days around water sampling periods. The top and bottom of each box-plot represent 75\% (upper quartile) and 25\% (lower quartile) of all values, respectively. The horizontal line is the median. The ends of the whiskers represent the 10$^{th}$ and 90$^{th}$ percentiles. Outliers are represented by empty dots. The $^{40}$K baseline at about 40 kHz is represented as a dashed line. \textit{Lux} mRNA/bacterial rRNA ratio are presented as part of active bioluminescent bacteria over the total active bacteria.}
\label{boxplotbiolu}
\end{figure} 

\subsection*{Conclusion}

The aim of this work is to describe total prokaryotic communities in the deep Mediterranean Sea, and to focus on bioluminescence activity conjointly to the detection of bioluminescent bacteria. During this survey, stable hydrological and physico-chemical conditions were observed with no input of water masses from the surface. In this context, prokaryotic community appeared relatively stable over the year. On a quantitative and qualitative aspect, \textit{Chloroflexi} and \textit{Proteobacteria} appeared as the most active members. In consequence, bioluminescence from eukaryotic organisms was limited due to the absence of mechanical stimulation and the identification of the bacterial part in light emission was probably easier to determine. Moreover, \textit{lux}F genes and transcripts were detected all over the year demonstrating the presence and activity of bioluminescent bacteria affiliated to \textit{Photobacterium} genus.\\

This study increases the interest to use similar approach during high bioluminescence activity events already detected in real-time by the ANTARES observatory in 2010 and 2009 \citep{tamburini2013, martini2013}. On the one hand, short sampling cruises should be managed during high bioluminescence events detection to compare both prokaryotic communities and bioluminescent bacteria quantification and activity. On the other hand, as continuous record of bioluminescence emission is already done by real-time data from the ANTARES cabled observatory. Automatic sampler for prokaryotic sampling might be an innovative tool to follow, understand and, at the end, predict extreme bioluminescence events in the deep sea.\\

%A deep Environmental Sampling Profiler (deep-ESP) is developed at the MBARI to reach such aim \citep{ussler2013}. This instrumentation should be adapted for abyssal environment and specific bacterial organisms detection as bioluminescent ones for example.\\

\textbf{Acknowledgements}\\
This work has been founded by the EC2CO-BIOLUX program. SM was granted a MENRT fellowship (Ministry of Education, Research and Technology, France). Authors thank N. Garcia, P. Raimbault for the nutrients analysis, B. Charrière for the DOC analysis, D. Lefèvre for the environmental data availability, S. Isart for her laboratory work, the crew of the N/O Thetys and the CASCADE-team. Authors thank the Collaboration of the ANTARES deep-sea observatory for providing
time series data.\\

%\newpage
%\section{Supplementary data}

%To support an increase in bacterial production, respiration rate was recorded in the deep sea using a newly developed system to sample prokaryotic oxygen consumption, the IODA $_{6000}$. Figure \ref{resp_anne} is the time series from 2009 to 2012 recorded at the ANTARES station.\\

%\begin{figure}[!h]
%\linespread{1} 
%\centering
%\includegraphics[width=12cm]{resp_anne.pdf}
%\caption[This dataset has been sampled using a new innovative tool, the \textit{in situ} Oxygen Dynamics Auto-sampler (IODA$_{6000}$).]{Supplementary data. This dataset has been sampled using a new innovative tool, the \textit{in situ} Oxygen Dynamics Auto-sampler (IODA$_{6000}$). This apparatus has been developed at laboratory to measure \textit{in situ} dissolved oxygen. The IODA$_{6000}$ is an auto-sampler with an oxygen optode fitted within an incubation chamber to measure biological activity and an external oxygen optode that measures \textit{in situ} O$_2$ concentration. For more information see \cite{robert2012}. These results are time series of prokaryotic respiration ($\mu$mol O$_2$ dm$^3$ d$^{-1}$) measured by IODA$_{6000}$ at 2,000 m depth at the ANTARES station. The prokaryotic respiration shows an increase in March 2011 enhancing the hypothesis of prokaryotes bloom in the deep sea after the spring bloom in chlorophylle-\textit{a} at the surface.}
%\label{resp_anne}
%\end{figure}

\section{Conclusions}

In Chapter 2 and 3, bioluminescence and new water masses events were related (see \citealp{tamburini2013, martini2013b}) in 2009 and 2010 impacting the deep ANTARES station. In this Chapter 5, we focused on year 2011.The CASCADE cruise occurs from 1$^{st}$ to 30$^{th}$ of March 2011 in the Gulf of Lion and sampling data from this expedition, show that no deep convection occurs below 1,600 m depth. This weak convection does not lead to deep water formations influencing bioluminescence activity at the ANTARES observatory (2,000 m) meaning that no links between water masses and bioluminescence activity can be highlighted during this period. This survey was an opportunity to describe the ANTARES site characteristics and the prokaryotic community over the year within stable hydrological conditions.\\

Between October 2010 and March 2013, the IL07 instrumented line has been turned off. The bioluminescence-activity data presented in this work have been extracted from one of the PMTs located in the whole observatory. Contrarily to the ones located on the IL07, during too high bioluminescence-intensity events, these PMTs are turned off. Such technical constraint induces gaps in time series record. However, these periods of high bioluminescence activity are shorter than those observed in 2009 and 2010 and does not occur during the sampled period.\\

The bioluminescence activity continuously recorded at the deep ANTARES station during this survey is relatively low and does not show high and long-time variation in bioluminescence activity in 2011, as previously observed by \cite{tamburini2013} and \cite{martini2013b}. To conclude from this survey, without deep-sea convection or cascading phenomena, bioluminescent bacteria have been estimated at about 0.1 to 1\% with high rate of active cells compared to total prokaryotes. Moreover, bioluminescent bacteria can not be excluded as a potential contributor of high level of bioluminescence activity already described in the deep sea. Sampling during high bioluminescence events are still needed to describe the bacterial role during high bioluminescent activity detected at the ANTARES station. Thanks to the real time data acquisition on the ANTARES observatory, adapted water sampling must be adapted to reach that goal in further investigations.\\

As perspective work, from this Chapter 5, firstly, the \textit{lux} genes analyzes might be improved. \textit{Lux}F gene was the most efficient at laboratory and has been used to detect bioluminescent bacteria. The \textit{lux}F function is identified for all meso and bathypelagic species involving a major role for deep-bioluminescent-species but with no clear identification. In further investigations, \textit{Lux}A primers coding for two sub-units of the bacterial luciferase has been used however the results were not decisive to improve our results robustness.\\

Moreover, during this survey, seasonal spring bloom has been observed in March 2011, at the surface of the ANTARES station, using chlorophylle-\textit{a} observations.\\

\begin{figure}[!h]
\linespread{1} 
\centering
\includegraphics[width=14cm]{chlorophylle.pdf}
\caption[Chlorophyll-\textit{a} time series recorded at the ANTARES station from satellite data, data acquisition.]{Chlorophyll-\textit{a} time series recorded at the ANTARES station from satellite data, data acquisition in collaboration with F. D'Ortenzio. Grey bars represent the discrete water sampling.}
\label{chlorophyll}
\end{figure}

Such bloom is already known to involve an increase in POC and DOC sinking to the deep sea. Indeed, it is known that 1 to 40\% of of the photosynthetically fixed carbon is exported into the dark realm where it is remineralized \citep{ducklow2001} by prokaryotes. Such seasonal spring blooms could involve higher abundance of bioluminescent bacteria fueling the deep sea and, by the end, an increase in bioluminescence activity. Using a suitable sampling survey, this phenomenon has to be distinguished from newly formed deep water events. However, to reach such information, it is necessary to sample after the occurrence of such bloom, and at high frequency, DOC, POC as well as prokaryotic communities and bioluminescence activity. 

%To estimate downward flux reaching the deep sea, the Martin model \citep{martin1987} is commonly used to estimate the organic carbon flux toward the ocean's interior following the form\string: \\
%$F=F_{100} \times (z/100)^{-b}$\\
%with $F_{100}$ the particulate organic flux at 100 m $z$ the depth and $b$ the particles transfer efficiency.\\  

%\begin{figure}[!h]
%\linespread{1} 
%\centering
%\includegraphics[width=13cm]{chap4-summary.pdf}
%\caption{Schematic representation of carbon input (POC and DOC) to the deep sea after a phytoplanktonic bloom occurring at the surface. Yellow boxes represent light emition from bioluminescent bacteria. This situation probably happens in 2011 during our prokaryotic survey with low bioluminescence emission.}
%\label{tambu2003}
%\end{figure}